src/univariate/discrete/betabinomial.jl

```diff
@@ -0,0 +1,64 @@
+immutable BetaBinomial <: DiscreteUnivariateDistribution
+    n::Int
+    α::Float64
+    β::Float64
+
+    function BetaBinomial(n::Real, α::Real, β::Real)
+        @check_args(BetaBinomial, n >= zero(n) && α >= zero(α) && β >= zero(β))
+        new(n, α, β)
+    end
+end
+
+@distr_support BetaBinomial 0 d.n
+
+function insupport(d::BetaBinomial, x::Real)
+    0 <= x <= d.n
+end
+#### Parameters
+
+params(d::BetaBinomial) = (d.n, d.α, d.β)
+#### Properties
+
+mean(d::BetaBinomial) = (d.n * d.α) / (d.α + d.β)
+function var(d::BetaBinomial)
+    n, α, β = d.n, d.α, d.β
+    numerator = n*α*β * (α + β + n)
```

Comment:
Minor nit: might be nice to use consistent spacing between the arguments to infix operators. Not worth spending much time on, but it's always nice to have whitespace used systematically.